Day 12/Guess-My-Number.py

```diff
@@ -0,0 +1,49 @@
+from art import logo
+import random
+
+def check_guess(guess, target):
+    if guess < target:
+        print("Too low.")
+        return False
+    elif guess > target:
+        print("Too high.")
+        return False
+    else:
+        print(f"You got it. The answer was {target}.")
+        return True
+
+def guessing_game():
+    print(logo)
+    print("Welcome to the Number Guessing Game!")
+    print("I'm thinking of a number between 1 and 100.")
+
+    random_number = random.randint(1, 100)
+    print(f"Psst, the correct answer is {random_number}.")
+
+    total_tries = 0
+    level = input("Choose a difficulty: Type 'easy' or hard': ").lower()
+
+    if level == "easy":
+        total_tries = 10
+    else:
+        total_tries = 5
```

Comment:
You can set the values for `total_tries` at each level as a constant since it should not change within the code.

Reply:
Resolved.